### DISCUSSION TITLE
Does this also work on a 3rd gen Lenovo Carbon X1?

### DISCUSSION THREAD

**Discussion by Atten007** [tylernguyen/x1c6-hackintosh#142] [Q&A] [answered]
Hi folks,

I want to buy a laptop for my new school where I‘m going to be arrive after the summer holidays. This laptop is a Lenovo X1 Carbon (3rd generation) with an Intel core i7 CPU and a 240GB SSD. I just wanted to quickly ask whether this also works with a Lenovo Carbon X1 of the 3rd generation.

Sincerely yours,
Atten007😊

**Comment by ajhutton** [comment, marked as answer]
Try this one or others for the T450s which is more similar to the X1C3
GitHub - CLAY-BIOS/Lenovo-ThinkPad-T450s-Hackintosh-OpenCore:
此仓库可以让你在macos中使用ThinkPad T450s T450 X250 X1C3的所有功能(不包括指纹)，且99%接近于白苹果的完美黑苹果。
<https://github.com/CLAY-BIOS/Lenovo-ThinkPad-T450s-Hackintosh-OpenCore>

On Sun, Aug 1, 2021 at 10:20 PM Atten007 ***@***.***> wrote:

> Hi folks,
>
> I want to buy a laptop for my new school where I‘m going to be arrive
> after the summer holidays. This laptop is a Lenovo X1 Carbon (3rd
> generation) with an Intel core i7 CPU and a 240GB SSD. I just wanted to
> quickly ask whether this also works with a Lenovo Carbon X1 of the 3rd
> generation.
>
> Sincerely yours,
> Atten007😊
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tylernguyen/x1c6-hackintosh/discussions/142>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAL3TUIPD354RRYTLPKNOFTT2U7JJANCNFSM5BLDNHQA>
> .
>


-- 
Thanks,
Alex

**Comment by tylernguyen** [comment]
@Atten007 

As stated by @ajhutton my EFI is not designed for the x1c3. You'll be better off forking an EFI built specifically for the x1c3 or any similarly spec variant.
